_layouts/community_tutorial.html

```diff
@@ -0,0 +1,79 @@
+---
+layout: default
+---
+
+<h1>{{ page.bioschemas.name }}</h1>
+<blockquote>
+  You are in the community-based tutorials space <br/>
+  {{ page.bioschemas.description }}
+</blockquote>
+{% include community-tutorial-navigation.html %}
+<div class="row">
+  <div class="col-md-8">
+    {{ content }}
+  </div>
+  <div class="col-md-4 right-column">
+    <script>
+      $(document).ready(function () {
+        $('#toc').toc({ minimumHeaders: 2, listType: 'ul', showSpeed: 0, headers: 'h2, h3', title: 'Table of Contents' });
+      });
+    </script>
+    <div id="toc" class="bg-light shadow p-3 rounded-3 my-4"></div>
+    <div class="bg-light shadow p-3 rounded-3 my-4">
+      {% if page.bioschemas.keywords %}
+      <p><strong>Keywords:</strong> {{page.bioschemas.keywords}}</p>
+      {% endif %}
+      {% if page.bioschemas.about %}
+      <p><strong>Topics:</strong></p>
+      <ul>
+        {% for topic in page.bioschemas.about %}
+        {% for item in topic %}
+        <li><a href="{{item[1]}}">{{item[1]}}</a></li>
+        {% endfor %}
+        {% endfor %}
+      </ul>
+      {% endif %}
+      {% if page.bioschemas.audience %}
+      <p><strong>Audience:</strong></p>
+      <ul>
+        {% for item in page.bioschemas.audience %}
+        <li>{{item.name}}</li>
+        {% endfor %}
+      </ul>
+      {% endif %}
+      {% if page.bioschemas.author %}
+      <p><strong>Authors:</strong> </p>
+      <ul>
+        {% for author in page.bioschemas.author %}
+        <li><a href="{{author.url}}">{{author.name}}</a></li>
+        {% endfor %}
+      </ul>
+      {% endif %}
+
+      {% if page.bioschemas.contributor %}
+      <p><strong>Contributors:</strong> </p>
+      <ul>
+        {% for contributor in page.bioschemas.contributor %}
+        <li><a href="{{contributor.url}}">{{contributor.name}}</a></li>
+        {% endfor %}
+      </ul>
+      {% endif %}
+
+      {% if page.bioschemas.license %}
+      <p><strong>License:</strong> {{page.bioschemas.license}}</p>
+      {% endif %}
+
+      {% if page.bioschemas.version %}
+      <p><strong>Version:</strong> {{ page.bioschemas.version }}</p>
+      {% endif %}
+
+      {% if page.bioschemas.dateCreated %}
+      <p><strong>Created:</strong> {{ page.bioschemas.dateCreated | date_to_long_string }}</p>
+      {% endif %}
+
+      {% if page.bioschemas.dateModified %}
+      <p><strong>Last Modified:</strong> {{ page.bioschemas.dateModified | date_to_long_string}}</p>
+      {% endif %}
+    </div>
+  </div>
+  {% include community-tutorial-navigation.html %}
```

Comment:
Same here. Just my opinion!

Reply:
@bedroesb if you see something is still missing, please go ahead and make the modifications

Reply:
@ljgarcia I already made my modifications, thanks :)